src/tests/createHeadlessForm.test.js

```diff
@@ -1839,6 +1863,16 @@ describe('createHeadlessForm', () => {
         ).resolves.toEqual(assertObj);
       });
     });
+    describe('and maximum is set to zero', () => {
```

Comment:
question: This new test is great, do you plan to keep the one added in previous commit too? I'd say just one merging all assertions is better. what do you think?

Reply:
Woops, I just forgot to get rid of the old test cases.

Do you have a ```suggestion on what other cases I could add here?

Reply:
I'd sy that the cases you wrote in the original test (0, 10, -11 5) are perfect. Just move them here and rename the description to `it("shows the correct validation")`